src/config.ts

```diff
@@ -22,6 +22,29 @@ export const CIE_IDP_IDENTIFIERS = {
     "xx_servizicie_test",
 };
 
+export const IDP_NAMES: Record<string, string | undefined> = {
```

Comment:
```suggestion
export const IDP_NAMES: Record<
  keyof typeof CIE_IDP_IDENTIFIERS | keyof typeof SPID_IDP_IDENTIFIERS,
  string | undefined
> = {
```
i suggest to add this as the type for the keys, so we can be sure all keys belong to the variables defined above: `SPID_IDP_IDENTIFIERS` and `CIE_IDP_IDENTIFIERS`.
also when adding a new identifier, compilation step will return an error if it's not present in this new `IDP_NAMES` object

Reply:
Nice catch. Fixed in d5ff63f with some rework.
The new `Issuer` type can be used by the `io-backend` to validate the SpidUser.